faraday/proxy.py

```diff
@@ -581,7 +581,12 @@ def main():
 
     # Associate serial ports with callsigns
     # global units
-    units = callsign2COM()
+    try:
+        units = callsign2COM()
+
```

Comment:
Fixed in https://github.com/FaradayRF/Faraday-Software/pull/193/commits/5db3383d59ad59ebe48530e1cf210c0b5c2a5334